plugins/providers/metabase/client.go

```diff
@@ -18,6 +19,7 @@ import (
 type MetabaseClient interface {
 	GetDatabases() ([]*Database, error)
 	GetCollections() ([]*Collection, error)
+	GetGroups() ([]*Group, map[string][]map[string]interface{}, map[string][]map[string]interface{}, error)
```

Comment:
lets have concrete types for this method.

Reply:
Can you also name these parameters.

Reply:
no, these are return values